src/main/java/com/ndgl/spotfinder/domain/post/controller/PostApiSpecification.java

```diff
@@ -85,33 +86,47 @@ RsData<Void> deletePost(
 
 	@Operation(
 		summary = "전체 포스트 조회",
-		description = "요청한 사이즈만큼 최신순으로 조회"
+		description = "요청한 사이즈만큼 최신순으로 조회",
+		security = {@SecurityRequirement(name = "JWT")}
 	)
 	RsData<SliceResponse<PostResponseDto>> getPosts(
-		SliceRequest sliceRequest
+		SliceRequest sliceRequest,
+		@Parameter(hidden = true) Principal principal
 	);
 
-	@Operation(summary = "포스트 1건 조회")
+	@Operation(
+		summary = "포스트 1건 조회",
+		security = {@SecurityRequirement(name = "JWT")}
+	)
 	RsData<PostDetailResponseDto> getPost(
-		@Parameter(description = "게시물의 ID") Long id
+		@Parameter(description = "게시물의 ID") Long id,
+		@Parameter(hidden = true) Principal principal
 	);
 
 	@Operation(
 		summary = "사용자가 작성한 포스트 목록 조회",
-		description = "요청한 사이즈만큼 최신순으로 조회"
+		description = "요청한 사이즈만큼 최신순으로 조회",
+		security = {@SecurityRequirement(name = "JWT")}
 	)
 	RsData<SliceResponse<PostResponseDto>> getPostsByUserId(
 		@Parameter(description = "사용자의 ID") Long userId,
-		SliceRequest sliceRequest
+		SliceRequest sliceRequest,
+		Principal principal
```

Comment:
```suggestion
		@Parameter(hidden = true) Principal principal
```
Principal은 가려주는게 좋아보입니다. (다른 도메인도 마찬가지!)